DemoServer/AdditionalLanguages/nodejs/demo/queries/projectingIndividualFields/index.js

```diff
@@ -18,21 +18,6 @@ async function run () {
 
     //region Step_3
     const projectedResults = await projectedQuery.all();
-    // [
```

Comment:
I would have left this since it shows the shape of the results. There's no class defined here like in Java or C# which describes what the fields are, so this comment serves as a description of how do the results look like.

Reply:
ok - added :)